acceptance/lib/puppet/acceptance/classifier_utils.rb

```diff
@@ -82,48 +82,6 @@ def create_group_for_nodes(nodes, group_hash)
         return group_uuid
       end
 
-      # Locates and then updates the 'PE MCollective' group to disable it for
-      # all agent nodes. A teardown is registered to restore the 'PE
```

Comment:
I think this is still necessary to be able to run Puppet's tests in a PE environment until we have a PE with MCO bits ripped out.

Reply:
Yeah, I wasn't sure about that one when removing it so thank you for reviewing that portion. Should I leave that in for now?

Reply:
Yes. We should file a ticket to remove this bit, and block that on the PE MCO removal work